src/main/java/net/fabricmc/loom/configuration/providers/minecraft/MinecraftProvider.java

```diff
@@ -58,9 +59,14 @@ public abstract class MinecraftProvider {
 	private BundleMetadata serverBundleMetadata;
 
 	private final Project project;
+	private final Property<Boolean> canMergeJars;
 
 	public MinecraftProvider(ConfigContext configContext) {
 		this.project = configContext.project();
+		this.canMergeJars = this.project.getObjects().property(Boolean.class).convention(this.project.provider(() -> {
+			return getVersionInfo().isVersionOrNewer("2012-07-25T22:00:00+00:00" /* 1.3 release date */);
+		}));
+		this.canMergeJars.finalizeValueOnRead();
```

Comment:
Its likely cleaner to move this into provide(), likely after downloading the jars. Maybe just have a nullable `Boolean` and throw if its null in canMergeJars(), there is no need to use a Gradle `Property` here.

Reply:
I'll move it to just after the version info has been downloaded - that's the earliest it could be done